sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -54,19 +54,15 @@
 #include "math/pprz_geodetic_int.h"
 #include "math/pprz_isa.h"
 
+#ifndef INS_BARO_AGL_OFFSET
```

Comment:
oh, maybe the name isn't very good but it is supposed to be the initial baro offset, it is currently actively estimated in the vff but it should be possible for the user to define a static offset value of the baro from the ground for eg when the vehicle has long legs or there is some large physical offset of the barometer from the ground. This is also used when resetting the altitude, when reset the baro altitude doesn't go to zero but is reset to this static value. Was useful to us recently when using a quad with long landing legs.